addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6544,7 +6544,9 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
     backgroundType: function (previewMode, widgetValue, params) {
         this.$target.toggleClass('o_bg_img_opt_repeat', widgetValue === 'repeat-pattern');
         this.$target.css('background-position', '');
-        this.$target.css('background-size', '');
+        widgetValue !== 'repeat-pattern'
+            ? this.$target.css('background-size', '')
+            : this.$target.css('background-size', '100px');
```

Comment:
```suggestion
this.$target.css('background-size', widgetValue !== 'repeat-pattern' ? '' : '100px');
```

Reply:
@can-odoo why not have made this Pr for version 14?

Reply:
> Hello @bvr-odoo Yes, I'll close this one and make a new PR for version 14.

Could you retarget this PR to V14 instead. We don't have to recreate a new PR (lose review history, etc.) if we just change the parent branch... Thanks

Reply:
Done.